File: sprint4/C++/lesson1code2.cpp
#include <iostream>
#include <map>
#include <string>

int main() {
    std::map<std::string, std::map<std::string, int>> pagesVisitStats {
        {"https://timofey.me", {{"main", 2}, {"contacts", 3}}},
        {"https://fruit-shop.ru", {{"apples", 12}, {"plums", 10}, {"pears", 20}, {"contacts", 15}}}
    };
    
    auto siteStats = pagesVisitStats["https://timofey.me"];
    return 0;
}

File: sprint4/kotlin/lesson1code2.kt
// Словарь "статистика посещения страниц"
val pagesVisitStats = mapOf(
    "https://timofey.me" to mapOf(
        "main" to 2,
        "contacts" to 3
    ),
    "https://fruit-shop.ru" to mapOf(
        "apples" to 12,
        "plums" to 10,
        "pears" to 20,
        "contacts" to 15
    )
)

val siteStats = pagesVisitStats["https://timofey.me"]
